src/devworkspace/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I've thought of moving this file into `src/services/helpers/devworkspace.ts`. WDYT?